docs-client/scripts/serve.ts

```diff
@@ -42,10 +42,13 @@ async function historyFallback(ctx: any, next: any) {
 }
 
 const proxyPort = process.env.PROXY_PORT || '8080';
+const proxyDocPrefix = process.env.PROXY_DOC_PREFIX || '/docs';
 
 const proxier = proxy('/', {
   target: `http://127.0.0.1:${proxyPort}`,
   changeOrigin: true,
+  logs: process.env.WEBPACK_SERVE,
```

Comment:
I wonder it is good or not. 🤔

Reply:
!!process.env.WEBPACK_SERVE would make it explicitly boolean for clarity.

Though this is only used in development so `true` would be fine too

Reply:
Thanks! it looks better. 👍